### DISCUSSION TITLE
Being able to toggle interface elements on/off

### DISCUSSION THREAD

**Discussion by the-baaron** [alefragnani/vscode-bookmarks#385] [Ideas]
Personally, I don't need to see the file name, or the `(Ln 50, Col 27)` thing. I just want a the names of my bookmarks. I understand it could be useful for other people tho. Could this be something to toggle on/off? Like a "clean design" and a "complete design"?

![Frame 1](https://user-images.githubusercontent.com/6582045/107617782-cac4a300-6c50-11eb-813e-cb39dd7d889b.png)

**Comment by alefragnani** [comment]
Hi @Ronald-Baars ,

There is an open issue, requesting for this #143 . There is also #145 with a broader idea (customize the preview content).

Feel free to upvote, and/or contribute with more details, if you see something that could be improved.

Hope this helps
